flaps/flaps_machines.go

```diff
@@ -263,13 +226,12 @@ func (f *Client) ListActive(ctx context.Context) ([]*api.Machine, error) {
 // Destroyed machines and console machines are excluded.
 // Unlike other List functions, this function retries multiple times.
 func (f *Client) ListFlyAppsMachines(ctx context.Context) ([]*api.Machine, *api.Machine, error) {
-	var allMachines []*api.Machine
+	allMachines := make([]*api.Machine, 0)
```

Comment:
The `var allMachines []*api.Machine` form is generally preferred if we're making a slice of an unknown size.